static/js/util/profile_edit.js

```diff
@@ -81,3 +104,44 @@ export function saveAndContinue(next) {
     browserHistory.push(next)
   });
 }
+
+export function editProfileObjectArray (arrayName, blankEntry, formCallback) {
+  const { updateProfile, profile } = this.props;
+  if ( profile[arrayName] ) {
+    let editForms = profile[arrayName].map( (entry, index) => (
+      formCallback.bind(this)(entry, index)
```

Comment:
Instead of `bind(this)(entry, index)` you can use `call(this, entry, index)`. However, why not `bind(this)` in the calling function instead? That would allow any function that returns a `ReactElement` to be used here